### DISCUSSION TITLE
Translations still not validated

### DISCUSSION THREAD

**Discussion by robertrub** [NightscoutFoundation/xDrip#2994] [General]
Hi

There were some errors in the French translations that I had corrected. 6 months later, they are still not validated. Am I wasting my time trying to help ???

![Screenshot_20230810_173502_DuckDuckGo.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/1b0c92f7-fbc9-47a0-9850-b428634de6c4)

**Comment by Navid200** [comment]
I don't know much about Crowdin.
If I'm not mistaken, each language has approvers.
If you have made a suggestion that has been rejected, ,there should be an explanation why.
If you have made a suggestion that has not been responded to and it has been quite a while, you can contact the managers and consult with them.

But, please note that all of this is done by volunteers.
So, statements like "am I wasting my time" may not be very productive.

**Reply by robertrub** [reply]
My message was to put forward something not working. Not start a fight.

I'm a volunteer too and spend huge amounts of time (much more for AAPS than here but try to help with translations and bugs here too). My hours are limited as yours are for sure. If I'm to spend time helping, I'd like it to be useful. If help is not acted upon, I just need to know it.
